scripts/integration-tests/e2e-create-react-app.sh

```diff
@@ -21,11 +21,20 @@ cd tmp/create-react-app || exit
 #                                   TEST                                       #
 #==============================================================================#
 
+# !!! WARNING !!!
```

Comment:
maybe a useless github action. but could be cool to comment/uncomment code to when a github issue is merged!